digits/extensions/data/interface.py

```diff
@@ -109,3 +109,27 @@ def itemize_entries(self, stage):
         this function in no particular order
         """
         raise NotImplementedError
+
+    @staticmethod
+    def can_explore():
```

Comment:
Also, when all the view extensions are converted, then the can_explore might just go away.